test/k8sT/manifests/etcd-deployment.yaml

```diff
@@ -4,7 +4,7 @@ metadata:
   name: "stateless-etcd"
 spec:
   ports:
-  - port: 2379
```

Comment:
This can still be `2379`

Reply:
@aanm I had the test failing due to port collision with the port 2379 whenever `stateless-etcd` and `etcd-k8s1` were deployed on the same node (`k8s1`). Would appreciate elaboration on how this can still be 2379? Force deployment onto `k8s2` instead?

Reply:
Or do you mean that the `spec` can say `port: 2379` even if the actual listening port is different due to the aforementioned conflict with `etcd-k8s1`?

Reply:
> Or do you mean that the `spec` can say `port: 2379` even if the actual listening port is different due to the aforementioned conflict with `etcd-k8s1`?

Yes, this

Reply:
@aanm Isn't it still better to have the spec have the port we actually use?

Reply:
So I don't get the benefit of leaving the port as 2379...

Reply:
@jrajahalme there's no major benefit, I was just pointing out that this port number does not reflect the port number used on lines 44 and 45 in this file.